gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -170,7 +170,9 @@ public void onCommandsSelected(CommandIndex path) {
   }
 
   private void updatePipelines() {
-    if (models.resources.isLoaded() && models.commands.getSelectedCommands() != null) {
+    if (!models.resources.isLoaded()) {
+      loading.startLoading();
+    } else if (models.commands.getSelectedCommands() != null) {
       loading.startLoading();
```

Comment:
Here `loading.startLoading()` is on both branches, and that may makes sense to make the logic clearer, but do you think this could be refactored?

Reply:
The flow in FramebufferView.java looks great, show hinting message when `models.commands.getSelectedCommands() == null`, and if all check passes, the else clause lead the following main part.